openedx/features/course_experience/static/course_experience/js/CourseOutline.js

```diff
@@ -4,7 +4,7 @@ import { keys } from 'edx-ui-toolkit/js/utils/constants';
 
 // @TODO: Figure out how to make webpack handle default exports when libraryTarget: 'window'
 export class CourseOutline {  // eslint-disable-line import/prefer-default-export
-  constructor() {
+  constructor(visualProgressTurnedOn) {
```

Comment:
`visualProgressTurnedOn` should really be something like `expandedCourseOutlineEnabled` now, right?